mopidy_local/www/index.html

```diff
@@ -1,43 +1,54 @@
 <!doctype html>
 <html>
   <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Mopidy-Local</title>
+    <script type="text/javascript">
+      var file = "http://" + window.location.host + "/mopidy/mopidy.css",
+        link = document.createElement("link");
+      link.href = file;
+      link.type = "text/css";
+      link.rel = "stylesheet";
+      link.media = "screen,print";
+      document.getElementsByTagName("head")[0].appendChild(link);
+    </script>
```

Comment:
If the only reason to use JavaScript here is to interpolate `window.location.host`, I think this could just be changed to:

```suggestion
    <link rel="stylesheet" href="/mopidy/mopidy.css" type="text/css" media="screen, print">
```

As an added bonus, this will also work if HTTPS is used.

Reply:
Yes, that seems to work